forum/qa-plugin/report-reason/q2apro-flag-reasons-lang-de.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
a my obsługujemy inne wersje językowe niż polski?

Reply:
Nic mi o tym nie wiadomo. Jednakże jak wspominałem jest to przerobiony plugin gdzie już to tłumaczenie było, więc szkoda mi było to wywalać.